package.json

```
@@ -11,8 +11,10 @@
     "postinstall": "npm run postinstall -w graphql-mesh && patch-package && npm run generate:mesh:lock",
     "build:local:packages": "concurrently \"npm run pack -w directive-spl\" \"npm run pack -w inject-additional-transforms\"",
     "preinstall": "npm run build:local:packages",
-    "start": "npm start -w graphql-mesh"
-  },
+    "start": "npm start -w graphql-mesh",
+    "startmesh": "npm run startmesh -w graphql-mesh",
+	"serve": "npm run serve -w graphql-mesh"
```

```suggestion
    "serve": "npm run serve -w graphql-mesh"
```

Your IDE currently lacks support for EditorConfig; please activate it, to prevent indentation issues in the future.